src/nmap.test.js

```diff
@@ -337,6 +349,9 @@ describe('nmap', () => {
                     operating_system: null,
                     scripts: null,
                     state: 'open',
+                    ostype: null,
+                    conf: null,
```

Comment:
Shouldn't  there be a test with some defined values for ostype, conf and extrainfo?

Reply:
That would not really test much more because it would be a fixed value that I give through and expect to stay the same. There's no logic for these variables of any kind, I only expect them to be there. But for the prettier looks I could add something there ofc.